packages/core/src/app/customer/StripeGuestForm.tsx

```diff
@@ -169,11 +169,8 @@ const StripeGuestForm: FunctionComponent<StripeGuestFormProps & FormikProps<Gues
 
     return (
         <>
-            <div className="checkout-form">
-                <LoadingOverlay
-                    hideContentWhenLoading
-                    isLoading={ isStripeLoading }
-                >
+            <CustomerSkeleton isLoading={isStripeLoading}/>
```

Comment:
Same here.

Reply:
Hi @bc-peng, according to the previous discussion https://github.com/bigcommerce/checkout-js/pull/1177#discussion_r1100815666, 
this comment no longer applies, right?

Reply:
Just to confirm, you need the Stripe component to be there while loading, even it is `display:none`, don't you ?

Reply:
@bc-peng that's correct, same logic as the shipping component